posts/Yoann/2020-12-11-hibernate-search-6-0-0-Final.adoc

```diff
@@ -0,0 +1,295 @@
+allowing clearer mappings, especially when bridges are parameterized;
++
+For more information about bridges, see link:{hibernateSearchDocUrl}#mapper-orm-bridge[this section of the documentation].
+
+Backend-agnostic::
+* Field declarations do not involve any Lucene API.
+* However, bridges can still declare native fields when necessary,
+be it with link:{hsearch-doc-url-prefix}#backend-lucene-field-types-extension[Lucene]
+or link:{hsearch-doc-url-prefix}#backend-elasticsearch-field-types-extension[Elasticsearch].
+
+[[automatic-indexing]]
+=== Easier to configure, smarter automatic indexing
+
+Automatic resolution of reindexing requirements::
+* `@ContainedIn` is no longer needed: when using `@IndexedEmbedded` on an association,
+Hibernate Search 6 infers the inverse side of the association from Hibernate ORM metadata,
+which allows it to automatically reindex the class hosting the `@IndexedEmbedded` annotation
+when the target of the association changes.
+* When the inverse side of an association cannot be resolved,
+Hibernate Search 6 will report a mapping error on bootstrap,
+allowing you to detect risks of out-of-sync indexes early.
+
+Easy override of reindexing requirements::
+* You can opt out of automatic reindexing locally using
+link:{hsearch-doc-url-prefix}#mapper-orm-reindexing-reindexonupdate[`@IndexingDependency(reindexOnUpdate = ...)`].
+* You can declare the dependencies of a `@Transient`, indexed property using
+link:{hsearch-doc-url-prefix}#mapper-orm-reindexing-derivedfrom[`@IndexingDependency(derivedFrom = ...)`].
+
+Easy configuration of indexing synchronization::
+Some want more performance with asynchronous automatic indexing,
+others prefer data safety and immediate visibility of indexed documents with synchronous automatic indexing.
+With Hibernate Search 6, this can all be configured
+with a link:{hsearch-doc-url-prefix}#mapper-orm-indexing-automatic-synchronization[single configuration property].
+
+Smarter change processing::
+Hibernate Search 6 is smarter when processing changes on complex entity graphs.
++
+When a property changes in an entity that is indexed-embedded in multiple other entities,
+Hibernate Search 6 will only traverse associations to entities that are actually
+affected by the change, based on `@IndexedEmbedded(includePaths = ...)` and other metadata.
+
+[[schema-management]]
+=== More flexible schema management
+
+Manage the schema on statup::
+Just like Hibernate Search 5, Hibernate Search 6 can
+link:{hsearch-doc-url-prefix}#mapper-orm-schema-management-strategy[manage the schema for you on startup].
+Or not: if this gets in the way, it can be disabled.
+
+Manage the schema on demand::
+Unlike Hibernate Search 5, Hibernate Search 6 offers
+link:{hsearch-doc-url-prefix}#mapper-orm-schema-management-manager[dedicated APIs to trigger schema operations on demand].
+
+Manage the schema when mass indexing::
+The MassIndexer can drop and re-create the schema automatically:
```

Comment:
If we use the a camel cased `MassIndexer`, I think we should put it under the apostrophes.